solidity/test/LiquidityProtectionAverageRate.js

```diff
@@ -19,9 +19,10 @@ const StandardPoolConverterFactory = contract.fromArtifact('TestStandardPoolConv
 const StandardPoolConverter = contract.fromArtifact('TestStandardPoolConverter');
 const LiquidityProtectionSettings = contract.fromArtifact('LiquidityProtectionSettings');
 const LiquidityProtectionStore = contract.fromArtifact('LiquidityProtectionStore');
-const LiquidityProtection = contract.fromArtifact('TestLiquidityProtection');
+const LiquidityProtectionStats = contract.fromArtifact('LiquidityProtectionStats');
```

Comment:
I might be missing something, but it looks like you haven't added tests that actually test that the stats are being updated? Did you forget to commit `LiquidityProtectionStats.js`?

Reply:
No, I'm in the process of implementing it. BTW, the actual migration script (`stats_migration.js`) does a pretty thorough verification.

Reply:
It's mutually exclusive.

Reply:
Done.

Reply:
👍